lib/semantics/resolve-labels.cc

```diff
@@ -359,10 +370,12 @@ class ParseTreeAnalyzer {
             if (const auto *otherPointer{
                     std::get_if<parser::Name>(&optionalGenericSpec->u)}) {
               if (namePointer->ToString() != otherPointer->ToString()) {
-                errorHandler_.Say(currentPosition_,
-                    parser::MessageFormattedText{
-                        "INTERFACE generic-name (%s) mismatch"_en_US,
-                        namePointer->ToString().c_str()});
+                errorHandler_
+                    .Say(currentPosition_,
+                        parser::MessageFormattedText{
```

Comment:
I don't think the explicit MessageFormattedText constructor is necessary. Message has a constructor that can take the position, fixed text, and arguments.